tests/test_utils.py

```diff
@@ -35,3 +35,29 @@ def test_dagmc_renumbering():
     assert len(combined_model.volumes) == num_vol_exp
     assert max(combined_model.volumes_by_id.keys()) == max_vol_id_exp
     assert all(mat in mats_exp for mat in mats)
+
+
+def test_expand_list():
```

Comment:
This test looks great. Can you include a docstring for this test in a similar format to what we have for the other tests?